docs/en/operations/system-tables/tables.md

```diff
@@ -84,6 +84,8 @@ Columns:
 
 - `loading_dependent_table` ([Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md))) - Dependent loading table.
 
+- `index_size` ([Nullable](../../sql-reference/data-types/nullable.md)([UInt64](../../sql-reference/data-types/int-uint.md))) - Total size of all indexes in the table.
```

Comment:
This new field would duplicate what is already shown in `system.parts`. Let's _only_ change `information_schema.tables` (which is defined by the SQL standard and generally not very usable, it only exists for compatibility with 3rd party tools).